cypress/e2e/GUI/gui.cy.js

```diff
@@ -0,0 +1,43 @@
+/// <reference types="Cypress" />
+
+describe('Testando a aplicação EngageSphere', () => {
+
+  
+  beforeEach(() => {
+    // Define uma data e hora fixa
+    const fixedDate = new Date(2024, 6, 10);
+    cy.clock(fixedDate.getTime());
+    cy.visit('http://localhost:3000');
+  });
+
+  
+  it('Testando o componente Header', () => {
+    cy.get('h1').should('have.text', 'EngageSphere');
+    cy.get('[data-testid="table"] > :nth-child(1)').should('have.text', 'Hi there! It is now Wed Jul 10 2024.');
```

Comment:
Essa verificação não faz sentido, visto que o teste é o Header.

Reply:
Além disso, esse seletor não é nada legível.

Um seletor simples seria algo como o seguinte `cy.contains('p', 'Hi there! It is now Wed Jul 10 2024.')`